frontend/next.config.js

```diff
@@ -23,7 +23,10 @@ const nextConfig = {
     },
     {
       source: "/api/training/:path*",
-      destination: "http://127.0.0.1:8000/api/:path*",
+      destination:
+        process.env.ENVIRONMENT === "production"
+          ? "http://alb-785155211.us-east-1.elb.amazonaws.com/api/:path*" // note, this url changes every time you destroy/apply Terraform
```

Comment:
this is really good to know. So we need to update this every single time we make a terraform change?

Reply:
I think for now, we can table the problem of making the ALB url more predictable and not requiring manual update of next.config.js for a later iteration in favor of development velocity

Reply:
i agree, since currently prod isn't ready yet